src/d_main.c

```diff
@@ -255,9 +255,16 @@ void D_Display (void)
     case GS_LEVEL:
       if (!gametic)
         break;
+      if (automapactive && !automapoverlay)
```

Comment:
As suggested, I added the `&& !automapoverlay` check. However, it caused some issues, and the only solution I came up with was to modify another check below...

Reply:
Thank you, much better already!